include/openthread.h

```diff
@@ -1669,7 +1670,7 @@ ThreadError otSetMessageOffset(otMessage aMessage, uint16_t aOffset);
  * @sa otReadMessage
  * @sa otWriteMessage
  */
-int otAppendMessage(otMessage aMessage, const void *aBuf, uint16_t aLength);
+ThreadError otAppendMessage(otMessage aMessage, const void *aBuf, uint16_t aLength);
```

Comment:
As a quick aside not related to these changes, we might want to consider renaming the message functions to all have the prefix `otMessage*()`. This makes it more clear that the subject of the method is an `otMessage` object. `otAppendMessage()` reads like we are appending an `otMessage` onto something, which is confusing.

Again, this isn't a comment related to this commit, just a note for future reference.